pkg/operator/workload/util.go

```diff
@@ -0,0 +1,28 @@
+package workload
```

Comment:
I don't think this makes sense, appending a flag to force rollout to an error slice.
It would make more sense to either have a separate variable for the force rollout, or create a data structure that stores a force rollout bool and a slice of errors.

Reply:
The suggested change makes possible to track which component is triggering the "force rollout" scenario, and by leveraging the slice of errors, it becomes possible to identify the ones will need special handling. We could achieve the same results with a custom type, however, the suggested changes are following a [common golang pattern](https://blog.golang.org/go1.13-errors).